troposphere/static/js/components/modals/instance/InstanceResumeModal.jsx

```diff
@@ -25,38 +28,72 @@ export default React.createClass({
     // ------
     //
 
-    renderBody: function() {
+    renderBody: function(multiple, instances) {
+        let operativeNoun = multiple ? "these instances": "this instance",
+            instanceList = (<div/>);
+
+        if (instances) {
+            instanceList = (
+            <div>
+                <ul>
+                {instances.map((i) => {
+                    return (
+                    <li key={i.get("uuid")}>
+                        <strong>
+                            {`${i.get("name")}`}
+                        </strong>
+                        <ul>
+                            <li style={{listStyleType: "square"}}>
+                                {`started on: ${i.get("start_date")}`}
+                            </li>
+                        </ul>
+                    </li>
+                    );
+                 })}
+                </ul>
+            </div>
+            );
+        }
+
         return (
         <div>
+            {instanceList}
             <p>
-                {"Would you like to resume this instance?"}
+                {`Would you like to resume ${operativeNoun}?`}
```

Comment:
I didn't handle the "no instances" case because I don't believe a modal will get rendered if they're at any. 

I realize since we're filtering out from `selectedResources` that might be an ill-conceived notion.